lib/validation.rb

```diff
@@ -0,0 +1,40 @@
+require "active_support/core_ext"
+
+class Validation
+  class << self
+    def validate_blank(attribute, name)
+      if attribute.blank?
+        return { "#{name}": "Cannot be empty" }
+      end
+      nil
```

Comment:
What I might do instead is to have a shared `errors` variable that I push error message into when the condition is met. So the code would look like:

```rb
errors << { "#{name}": "Cannot be empty" } if attribute.blank?
```

And I won't have to worry about dealing with `nil`.